content/react/concepts/components/components.md

```diff
@@ -45,7 +45,20 @@ class ClassComponent extends React.Component {
 Function components are stateless and only use the `return` statement. No import statement necessary:
 
 ```jsx
-function FunctionComponent() {
+function FunctionComponent(props) {
+  return (
+    <div>
+      ...
+    <div>
+  )
+}
+```
+## Arrow function
```

Comment:
```suggestion

## Arrow Function

```

Let's use "Title Case" for the headers and don't forget to surround them with an empty line. 😃